src/Config/Haskell.hs

```diff
@@ -48,7 +51,11 @@ readSetting :: Scope -> Decl_ -> [Setting]
 readSetting s (FunBind _ [Match _ (Ident _ (getSeverity -> Just severity)) pats (UnGuardedRhs _ bod) bind])
     | InfixApp _ lhs op rhs <- bod, opExp op ~= "==>" =
         let (a,b) = readSide $ childrenBi bind in
-        [SettingMatchExp $ HintRule severity (head $ snoc names defaultHintName) s (fromParen lhs) (fromParen rhs) a b]
+        let unit = W (GHC.noLoc $ GHC.ExplicitTuple GHC.noExt [] GHC.Boxed) in
+        [SettingMatchExp $
+         HintRule severity (head $ snoc names defaultHintName) s (fromParen lhs) (fromParen rhs) a b
+        -- Todo : Replace these with "proper" GHC expressions.
```

Comment:
Or don't - happy to deprecate Haskell config at the same time.

Reply:
Interesting. I'd be happiest with that if it's feasible. Can you find a moment to talk to me about that or drop me a mail with a brief outline of what it would take?